samples/nrf5340/netboot/README.rst

```diff
@@ -1,88 +1,95 @@
 
 Overview
 ********
 
-The network core bootloader sample supports transporting an already verified and authenticated firmware upgrade from the application core flash to the network core flash, as well as performing flash protection.
+This sample supports transporting an already verified and authenticated firmware update from the application core flash memory to the network core flash memory.
+Also, this sample protects the flash memory areas allocated to both itself and the application running on the network core.
 
-This is accomplished by the following steps:
+The bootloader accomplishes these two objectives as follows:
 
-1. Lock the flash of the bootloader.
-     The bootloader sample locks the flash that contains the sample bootloader and its configuration.
+1. It locks the flash memory areas containing the bootloader sample and its configuration.
      Locking is done using the ACL peripheral.
      For details on locking, see the :ref:`fprotect_readme` driver.
+#. It performs any pending network core firmware update.
+     The sample calls the :ref:`subsys_pcd` library to inspect the SRAM region shared with the application core:
 
-#. Perform any pending network core firmware upgrade.
-     Invoke the :ref:`subsys_pcd` library to inspect a SRAM region shared with the application core.
-     If the application core has written an update instruction, copy the specified data range to the application partition on the network core.
-     Once the copy is done, compare the SHA of the data in the application partition against the SHA specified in the shared SRAM.
-     Communicate the result of the comparison to the application core through the shared SRAM.
-
-#. Lock the flash of the application.
-     Lock the flash that contains the application.
+     a. If the application core has written an update instruction, the bootloader copies the specified data range to the application partition on the network core.
+     #. Once the copy is done, the sample compares the SHA of the data in the application partition against the SHA specified in the shared SRAM.
+     #. It then communicates the result of the comparison to the application core through the shared SRAM.
+#. It then locks the flash memory areas containing the application.
      Locking is done using the ACL peripheral.
      For details on locking, see the :ref:`fprotect_readme` driver.
-
-#. Boot the application on the network core.
-     After possibly performing a firmware update, and enabling flash protection, the network core bootloader uninitializes all peripherals that it used and boots the application.
+#. It boots the application on the network core.
+     After possibly performing a firmware update, and enabling flash memory protection, the network core bootloader uninitializes all peripherals that it used and boots the application.
 
 Requirements
 ************
 
-  * |nRF5340DK|
+The sample supports the following development kits:
```

Comment:
It only supports the `nrf5340dk_nrf5340_cpunet` board, the table below correctly lists that build target as the only supported, but maybe this sentence should reflect that as well?

Reply:
good point, fixed